docs/DeveloperGuide.md

```diff
@@ -1,34 +1,726 @@
 # Developer Guide
 
-## Design & implementation
+## Introduction
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+`FridgeFriend` is an app for managing food in the fridge, optimised for use via a Command Line Interface (CLI).
+If you can type fast, `FridgeFriend` can track your cold or frozen groceries faster and easier than any other apps.
+It is written in Java, and has more than 4.3kLoC.
 
+## Contents
 
-## Product scope
-### Target user profile
+* [Introduction](#introduction)
+* [Design](#design)
```

Comment:
Maybe can add in a setting up the project section at the top of the DG to instruct people on how to set up the code in their computer , instead of putting it at the bottom under instructions for manual testing